compiler/lib-wasm/generate.ml

```diff
@@ -661,7 +661,11 @@ module Generate (Target : Target_sig.S) = struct
                 in
                 Memory.allocate ~tag:0 ~deadcode_sentinal:ctx.deadcode_sentinal l
             | Extern name, l -> (
-                let name = Primitive.resolve name in
+                let name =
+                  match name with
+                  | "caml_alloc_stack" when effects_cps () -> "caml_cps_alloc_stack"
```

Comment:
@vouillon, could this be moved to the wasm runtime somehow using conditional ? `(@if ..`

Reply:
See #1920